pnpm-lock.yaml

```diff
@@ -36,7 +36,7 @@ importers:
     devDependencies:
       '@graphprotocol/graph-cli':
         specifier: 0.61.0
-        version: 0.61.0(@types/node@20.10.3)(node-fetch@3.3.2)(typescript@5.3.3)
+        version: 0.61.0(@types/node@20.10.3)(node-fetch@3.3.2)(typescript@5.3.2)
```

Comment:
These version diffs (which appear unrelated to the example app you added) should probably be removed and might cause a conflict with main. Will circle back on this.